packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -398,6 +401,10 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
       });
     }
 
+    if (!this.props.hideTooltip && calloutProps!.isCalloutVisible) {
```

Comment:
Should this happen in the second render cycle in case optimize flag is set

Reply:
Done